test/bigtest/interactors/marc-field-protection/marc-field-protection.js

```diff
@@ -0,0 +1,12 @@
+import {
+  interactor,
+  isPresent,
+  count,
+} from '@bigtest/interactor';
+
+@interactor class MARCFieldProtection {
+  hasList = isPresent('#editList-marc-field-protection');
+  rowCount = count('#editList-marc-field-protection [class^="editListRow---"]');
+}
```

Comment:
Is it possible to change ids to data-test-... attributes ?

Reply:
This id is provided by `ControlledVocab` component, so I decided to use it instead of adding additional data-test attribute